UnityProject/Packages/packages-lock.json

```diff
@@ -106,7 +106,7 @@
       "url": "https://packages.unity.com"
     },
     "com.unity.scriptablebuildpipeline": {
-      "version": "1.19.6",
+      "version": "1.20.1",
```

Comment:
is this an unintended change?

Reply:
this is intended.

Reply:
oh wait, i thought this was this #9291 PR.
well either way it gets updated in that PR as well so its fine